scripts/generate.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Copyright 2015 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script to fetch latest swagger spec.
+# Puts the updated spec at api/swagger-spec/
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")
+CLIENT_ROOT=${SCRIPT_ROOT}/..
+
+[[ -d ${SCRIPT_ROOT}/bin ]] || mkdir ${SCRIPT_ROOT}/bin
+[[ -f ${SCRIPT_ROOT}/bin/swagger-codegen-cli.jar ]] || curl http://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.2.1/swagger-codegen-cli-2.2.1.jar > ${SCRIPT_ROOT}/bin/swagger-codegen-cli.jar
```

Comment:
Look like they do: https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen-maven-plugin/README.md

Only issue here is I am using an unofficial jar file right now in my bin folder that has my latest fix to codegen swagger-codegen#4050 If I can specify this in the maven plugin (to use the latest version from repo), I will change this to maven.